package/network/services/lldpd/files/lldpd.init

```diff
@@ -109,7 +124,7 @@ write_lldpd_conf()
 	config_get lldp_hostname 'config' 'lldp_hostname' "$(cat /proc/sys/kernel/hostname)"
 
 	local ifnames
-	get_config_cid_ifaces ifnames "interface"
+	get_interface_csv ifnames 'config' "interface"
```

Comment:
I wouldn't mix quoting.

Reply:
While I entirely agree with this sentiment and would like to change it, the entire rest of this file already uses mixed quoting. Where uci stuff is generally single quoted, and other shell and miscellaneous stuff seems double quoted, and my changes might otherwise look out of place. :/ 

Thanks for the review :)